js/main.js

```diff
@@ -0,0 +1,32 @@
+/* global data */
+
+const imgSrcChange = document.querySelector('.changed');
+const photoId = document.querySelector('.photoURL');
+const form = document.querySelector('form');
+
+photoId.addEventListener('input', (e) => {
+  if (e.target.value === imgSrcChange.setAttribute('src', e.target.value)) {
+    imgSrcChange.setAttribute('src', e.target.value);
+  } else if (e.target.value === '') {
+    imgSrcChange.src =
+      'https://i0.wp.com/tnpta.org/wp-content/themes/learnpro/framework/img/us-placeholder-square.jpg?resize=600%2C600&ssl=1';
+  }
+});
```

Comment:
No `if` statement. You should just be setting the `src` attribute of `imgSrcChange` directly